test/common/index.js

```diff
@@ -814,9 +814,10 @@ exports.getBufferSources = function getBufferSources(buf) {
 };
 
 // Crash the process on unhandled rejections.
-exports.crashOnUnhandledRejection = function() {
-  process.on('unhandledRejection',
-             (err) => process.nextTick(() => { throw err; }));
+const crashOnUnhandledRejection = (err) => { throw err; };
+process.on('unhandledRejection', crashOnUnhandledRejection);
```

Comment:
Was it intentional to drop the `process.nextTick()`?

Reply:
It was not. Do you think we should put it back?

Reply:
/cc @addaleax since you implemented it initially. Was there a reason to throw in the next tick? I can't see any difference on a local test with or without `process.nextTick`.

Reply:
@targos I don’t remember – I’d assume dropping it is okay.